packages/suite/src/components/suite/FiatValue/index.tsx

```diff
@@ -37,7 +37,10 @@ const FiatValue = ({
     ...props
 }: Props) => {
     const targetCurrency = fiatCurrency ?? props.settings.localCurrency;
-    const currentFiatRates = props.fiat.coins.find(f => f.symbol === symbol)?.current;
```

Comment:
since we know where the issue is, maybe it should be fixed in the source (connect? blockchain-link?)
which symbol was in upperCase here?

Reply:
`symbol`  was in upperCase in case of token transfer (because of lines 61:63 in `AmountDetails` in the old version).

If I just remove the unnecessary `toUpperCase()` from `AmountDetails`, the code works ok for now. 

I used `toLowerCase()` here in  `FiatValue` just to double-check this kind of error won't happen again.